lib/main.dart

```diff
@@ -31,7 +35,17 @@ import 'view/staff_welfare/your_welfare_view.dart';
 import 'view/view_templates/html_text_card_view_template.dart';
 
 void main() {
-  runApp(const MyApp());
+  // Add this, and it should be the first line in main method to ensure no crashes before runApp()
+  // Currently Crashlytics and SharedPreferences packages below use platform channels so need this before runApp()
+  WidgetsFlutterBinding.ensureInitialized();
+
+  // Crash Reporting configured for not reporting whilst doing Dev mode builds
+  CrashReporting();
```

Comment:
it looks quite strange just calling a constructor and then not doing anything with the object itself. Since CrashReporting has no instance properties can it just be a static function?

Reply:
I'll check

Reply:
So consolidating the comment about the constructor with this one: can really just be a static function, no class needed

Reply:
ok just a static function now.